### DISCUSSION TITLE
story-driven development using storyboard and solito

### DISCUSSION THREAD

**Discussion by karmatradeDev** [nandorojo/solito#98] [Q&A] [answered]
Hi, I usually build react apps but react native is honestly kinda different. I would like to do story-driven design which is building the storyboard components first and then build the screens and features. any ideas on how to use storyboard in solito? should I make a new react app in the apps folder our add to packages? kinda lost.

**Comment by nandorojo** [comment, marked as answer]
this might help you for inspiration: https://github.com/axeldelafosse/storybook-rnw-monorepo

**Reply by karmatradeDev** [reply]
thank you! worked perfectly i did:

cd apps

run yarn create next-app --typescript (named web-storybook)

updated tsconfig.json to :
`{
  "extends": "../next/tsconfig"
}`

run npx storybook init

goto project root package.json

add '  "storybook-web": "yarn workspace web-storybook run storybook" '  to scripts

run yarn run storybook-web 

worked perfectly
